### PR TITLE
Add missing allocate, capacity, pop for DS\PriorityQueue

### PR DESCRIPTION
**Pull request by tfedor**

### CODE REVIEW

File: ds/ds.php [hunk 1]
```diff
@@ -2463,6 +2479,15 @@ public function isEmpty(): bool {}
...
 
+        /**
+         * Removes and returns the value with the highest priority
+         * @link https://www.php.net/manual/en/ds-priorityqueue.pop.php
+         *
+         * @return mixed The removed value which was at the front of the queue.
+         * @throws UnderflowException if empty.
+         */
+        public function pop() {}
+
```
Comment (DmitryTronin) [inline ds/ds.php:2490]: Please run php-cs-fixer as described in readme to reformat the code. Or just remove whitespaces here, per failed tests: https://github.com/JetBrains/phpstorm-stubs/pull/1430/checks#step:5:8

**Review by DmitryTronin — CHANGES_REQUESTED** [review]
Also please update your project & push again to include the recent constants changes: https://github.com/JetBrains/phpstorm-stubs/pull/1431

**Comment by DmitryTronin** [comment]
Hi @tfedor. Could you please address the changes requested here if you have spare time?

**Comment by tfedor** [comment]
@DmitryTronin sorry for delay, should be fixed now

**Pull request merged**